### PR TITLE
Rename `OpenTF` to `OpenTofu`

### LINKED ISSUE #9712: Add OpenTofu (ex OpenTF)
### Brand Name

OpenTofu

### Website

https://opentofu.org/

### Popularity Metric

As of 2023-10-12,
- The Similarweb (global) rank is 334,602. See https://www.similarweb.com/website/opentofu.org
- The number of Github stars is 14.4k. See https://github.com/opentofu/opentofu/tree/main

### Official Resources for Icon and Color

svg (dark): https://github.com/opentofu/opentofu.org/blob/main/.github/logo-dark.svg
svg (light): https://github.com/opentofu/opentofu.org/blob/main/.github/logo-light.svg
website tailwind config: https://github.com/opentofu/opentofu.org/blob/main/tailwind.config.js

### Additional Comments

From [website](https://opentofu.org/):
> **Previously named OpenTF**, OpenTofu is a fork of Terraform that is open-source, community-driven, and managed by the Linux Foundation.

OpenTF simple-icon: https://github.com/simple-icons/simple-icons/blob/6d6ea814c3ed7487ad3b5abda246ce478c9a0ca6/icons/opentf.svg#L1

### PR DESCRIPTION
**Pull request by adamrusted**
opentofu

Closes #9712

**Similarweb rank:** [150,453](https://www.similarweb.com/website/opentf.org/#overview) (Old URL, new one not indexed yet)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Remove OpenTF icon, replaced with OpenTofu icon.

### CODE REVIEW

**Comment by elliotwutingfeng** [comment]
**New URL Similarweb rank:** [334,602](https://www.similarweb.com/website/opentofu.org/#overview)
The new URL now meets the criteria.

File: sdk.mjs [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
     .replace(/&#([0-9]+);/g, (_, num) => String.fromCodePoint(parseInt(num)))
     .replace(
       /&(quot|amp|lt|gt);/g,
-      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
```
Comment (PeterShaggyNoble) [inline sdk.mjs:122]:
```suggestion
      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Icon looks good, @adamrusted 👍🏻 Although there are some excess points in the corners of the eyes that you might be able to remove.

Also, is the change to `sdk.mjs` intentional?

File: sdk.mjs (hunk 1 above)
Comment (adamrusted) [inline sdk.mjs:122]:
```suggestion
      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
```

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**